core/src/test/scala/com/metamx/tranquility/test/TranquilizerTest.scala

```diff
@@ -166,8 +167,8 @@ class TranquilizerTest extends FunSuite with Logging
       newTranquilizer(beam, maxBatchSize, maxPendingBatches, lingerMillis).withFinally(_._1.stop()) {
         case (tranquilizer, desc) =>
           val (acked, dropped, failed) = doSend(tranquilizer, Seq("hey", "__drop__"))
-          assert(acked === 0, "acked (%s)" format desc)
-          assert(dropped === 2, "dropped (%s)" format desc)
+          assert(acked === 1, "acked (%s)" format desc)
+          assert(dropped === 1, "dropped (%s)" format desc)
```

Comment:
See the removed "This can actually be wrong" comment in Tranquilizer; the return value used to be wrong ("hey" was sent, "__drop__" was dropped, so acked should be 1 and dropped should be 1)